audio_service/README.md

```diff
@@ -345,23 +345,31 @@ Additionally:
 </manifest>
 ```
 
-2. Starting from Flutter 1.12, you will need to disable the `shrinkResources` setting in your `android/app/build.gradle` file, otherwise the icon resources used in the Android notification will be removed during the build:
+2. If you use any custom icons in notification, you should create `res/raw/keep.xml` as shown below, otherwise the icon resources will be removed during the build:
```

Comment:
I would start with the simple case first, assuming that most users are just using the default icons:

2. Create the file `android/app/src/main/res/raw/keep.xml` below to prevent the default notification icons from being stripped during the build process:

...

If you are using your own custom icons, you may selectively strip the default icons like this:

...

Reply:
icons from the library are not stripped by R8 so no keep rules needed, that's why "If you use any custom icons in notification" is in there

Reply:
this is by the way applies to the small notification too

Reply:
not using custom one -> no need for rules

Reply:
we should probably make it more clear, i will update this later